include/seqan3/argument_parser/detail/format_help.hpp

```diff
@@ -424,7 +422,7 @@ class format_help : public format_help_base<format_help>
     //!\brief Needed for correct formatting while calling different print functions.
     bool prev_was_paragraph{false};
     //!\brief Stores the relevant parameters of the documentation on the screen.
-    console_layout_struct layout;
+    static inline console_layout_struct const layout{};
```

Comment:
If it's static and const, it might as well be constexpr.

But this struct holds

```
            // Guess terminal screen width and set into layout.
            unsigned cols = get_terminal_width();
```

So the terminal width is set at compile time anyway?

Reply:
I guess I could make this constexpr but since the format is not I don't know if it is worth it